added more complete version of LUOV entry including benchmark files

chrysogonus opened the pull request:
I tried to create a more complete version of the already existing project of Henning by including benchmark files. The data can be extracted from the NIST Round 2 supporting documentation paper (https://github.com/WardBeullens/LUOV/blob/master/Supporting_Documentation/luov.pdf).

franziskuskiefer commented on signatures/LUOV/LUOV.yaml:
```diff
@@ -0,0 +1,19 @@
+problems:
+  - assumption: MQ
+sources:
+  - NIST Submission Paper Round 2 Version
```
Can you add a link for this one as well?

franziskuskiefer commented on signatures/LUOV/LUOV/impl/avx2-chacha8.yaml:
```diff
@@ -0,0 +1,4 @@
+name: avx2-chacha8
+comment: AVX2 optimized implemetation using Chacha8 as PRNG
```
Can you add links to the implementations?

dingens commented on signatures/LUOV/LUOV/bench/avx2-chacha8_LUOV-8-107-371_asus.yaml:
```diff
@@ -0,0 +1,8 @@
+platform: ASUS S410 Notebook PC with an Intel Core i5-8250U CPU @ 1.60 GHz
```
(@all files in `bench/`) I'd prefer if we named the files `kabylake` instead of `asus`, as that's the more relevant info.

dingens commented on signatures/LUOV/LUOV/bench/avx2-chacha8_LUOV-8-107-371_asus.yaml:
```diff
@@ -0,0 +1,8 @@
+    enc|sign: 3100000
+    dec|vrfy: 1500000
+  unit: cycles
+  comment: average of 1000 executions compiled with ”gcc − O3” with gcc version 7.3.0
```
(@all files in bench/) Not the most important thing but there's a typographic whatever dash not the ascii/shell minus one. I wonder how that happened, do you have you editor set to magicness=high or something? :D

dingens commented on signatures/LUOV/LUOV/impl/avx2-keccak.yaml:
```diff
@@ -0,0 +1,4 @@
+name: avx2-keccak
```
I strongly assume the keccak and chacha8 versions have different input and output. I would say they are different flavors then. At the very least, it's different parameter sets.

chrysogonus commented:
I've updated the entry and fixed some issues. Regarding @dingens request it seems that both PRNG flavours (keccak and chacha8) indeed work on the same parameters of the LUOV scheme. 
For more information see https://github.com/WardBeullens/LUOV/blob/master/Supporting_Documentation/luov.pdf

dingens commented:
Same parameters (in the sense of: sizes): yes. But: a) output will be different (there's different KAT files for both) b) timings will likely differ, as will code dependencies. That's the same situation as with many other schemes, and we've always been using separate flavors in those cases. You should be able to just copy the whole param/ dir over (just changing the `name:` line)
